docs/source/upgrade-2.0.rst

```diff
@@ -166,6 +166,395 @@ Rationale
 These existed to support defining coroutines in Python 2 and early versions of Python 3 before :term:`coroutine functions <coroutine function>`
 using the :keyword:`!async`\ /:keyword:`!await` syntax was added in Python 3.5.
 We no longer support versions of Python that don't support :keyword:`!async`\ /:keyword:`!await`,
-Python coroutines are noticeably faster than :deco:`cocotb.coroutine`'s implementation,
-and the behavior of :deco:`cocotb.coroutine` would have to be changed to support changes to the scheduler.
-For all those reasons the :deco:`cocotb.coroutine` decorator and generator-based coroutine support was removed.
+Python coroutines are noticeably faster than :deco:`!cocotb.coroutine`'s implementation,
+and the behavior of :deco:`!cocotb.coroutine` would have to be changed to support changes to the scheduler.
+For all those reasons the :deco:`!cocotb.coroutine` decorator and generator-based coroutine support was removed.
+
+
+********************************************************
+:class:`!BinaryValue` replaced with :class:`!LogicArray`
+********************************************************
+
+Change
+======
+
+:external+cocotb19:py:class:`~cocotb.binary.BinaryValue` and :external+cocotb19:py:class:`~cocotb.binary.BinaryRepresentation` were removed
+and replaced with the existing :class:`.Logic` and :class:`.LogicArray`.
+
+
+How to Upgrade
+==============
+
+* Change all constructions of :class:`!BinaryValue` to :class:`!LogicArray`.
+* Replace construction from :class:`int` with :meth:`.LogicArray.from_unsigned` or :meth:`.LogicArray.from_signed`.
+* Replace construction from :class:`bytes` with :meth:`.LogicArray.from_bytes` and pass the appropriate ``byteorder`` argument.
+
+.. code-block:: python
```

Comment:
I guess these code blocks under list items should be indented because they are belonging to their preceding list item instead of breaking up the list like a non-indented paragraph would also do?